.example.env

```diff
@@ -42,3 +42,8 @@ SMTP_HOST=
 SMTP_HOST_PASSWORD=
 SMTP_HOST_USER=
 SMTP_PORT=
+
+# Minio
```

Comment:
these values will also be dynamic on staging ,so I'm assuming these are defaults for local?